modules/gltf/gltf_document.cpp

```diff
@@ -2948,6 +2948,7 @@ Error GLTFDocument::_parse_meshes(Ref<GLTFState> state) {
 				} else {
 					Ref<StandardMaterial3D> mat3d;
 					mat3d.instantiate();
+					mat3d->set_meta("filler_id", vformat("@FILLER:%s:%s", import_mesh->get_name(), itos(j)));
```

Comment:
The rest looks good. I want to focus attention on the exact wording of this filler_id because we may have to live with it for a while.

Reply:
1. Do we use `@FILLER` style anywhere else? Like at and upper case filler.
2. Do we want to synchronize with the gltf material placeholder codes?
3. The double colons looks weird.
4. We've been migrating using meta for anything in editor. Don't have any ideas other than creating a property.

Reply:
### 1.

I used `import_id` as orientation.
https://github.com/godotengine/godot/blob/6f5704d86f95171ba8b6b2ac9f56e284c4d35d7a/editor/import/scene_import_settings.cpp#L146-L152

`@Filler` or `FILLER` is never used anywhere else

### 2.

Don't know how the placeholder codes work. (Every Material after the import need an identifier, even if the GLTF-file has zero materials)

### 3.

`_` and `-` are alerady used in other parts of the content and spaces feel wrong

### 4.

Don't know the import system enough to know what to change

### Ideas

Change meta to `"identifier"` (maybe `"identifier_id"` but id is already short for identifier)
Change content to `"%s@%s", import_mesh->get_name(), itos(j)`

The content is only visible in the `<file>.import`.

Current 
```
_subresources={
"materials": {
"@FILLER:Assets-f1737e9e675e596b2cfccf62dc0e906e_Cube002:0": {
"use_external/enabled": true,
"use_external/path": "res://Assets/Materials/Bullet.tres"
},
```

Idea
```
_subresources={
"materials": {
"Assets-f1737e9e675e596b2cfccf62dc0e906e_Cube002@0": {
"use_external/enabled": true,
"use_external/path": "res://Assets/Materials/Bullet.tres"
},
```